app/Controllers/javascriptController.php

```diff
@@ -75,7 +75,7 @@ public function nonceAction(): void {
 		$this->view->salt1 = sprintf('$2a$%02d$', FreshRSS_password_Util::BCRYPT_COST);
 		$alphabet = './ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
 		for ($i = 22; $i > 0; $i--) {
-			$this->view->salt1 .= $alphabet[random_int(0, 63)];
+			$this->view->salt1 .= $alphabet[mt_rand(0, 63)];
```

Comment:
This one was better with `random_int` as this one benefits from from a heavier cryptographically secure RNG.
Please do not change them in this PR, as they are unrelated to the topic of the PR